robotbase/src/main/java/com/systemmeltdown/robot/controls/GunnerControls.java

```diff
@@ -23,8 +31,13 @@
 
     public AxisThresholdTrigger m_rightTrigger;
     public AxisThresholdTrigger m_leftTrigger;
+    public JoystickButton m_DpadUp;
     public JoystickButton m_yButton;
     public JoystickButton m_xButton;
+    public Trigger m_aButton;
```

Comment:
May want to change the names of these buttons to represent that they need the DpadUp to be called. Something like m_aButtonAndDpadUp.